packages/logger/src/rld-inspect.ts

```diff
@@ -0,0 +1,203 @@
+import { match } from '@reatom/lens'
+import * as model from './rld-inspect-model'
+import { JSXElement } from '@reatom/jsx'
+import { Atom, AtomCache, AtomMut, Ctx, action, atom } from '@reatom/core'
+import styled from 'stylerun'
+// @ts-expect-error
+import { Inspector } from '@observablehq/inspector'
+import { t } from './t'
+
+export function RldInspect() {
+  const height = atom(400, 'height')
+  const graphWidth = atom(300, 'graphWidth')
+  const stateWidth = atom(300, 'stateWidth')
+
+  return t.div({
+    ...inspectStyles({}),
+    children: [
+      InspectView({
+        height,
+        width: graphWidth,
+        children: [InspectGraph()],
+      }),
+      match(
+        model.logSelected,
+        InspectView({
+          height,
+          width: stateWidth,
+          children: [InspectState()],
+        }),
+        t.div([]),
+      ),
+    ],
+  })
+}
+
+function InspectView({
+  height,
+  width,
+  children,
+}: {
+  height: AtomMut<number>
+  width: Atom<number>
+  resizeLeft?: (ctx: Ctx, to: number) => void
+  resizeRight?: (ctx: Ctx, to: number) => void
+  children: JSXElement[]
+}) {
+  return atom((ctx) =>
+    t.div({
+      ...viewStyles({}),
+      children: [
+        t.div({
+          ...viewHandleTopStyles({}),
+          children: [
+            t.div({
+              ...viewHorStyles({}),
+              children: [
+                t.div({
+                  ...viewContentStyles({}),
+                  style: `width: ${ctx.spy(width)}px; height: ${ctx.spy(
+                    height,
+                  )}px;`,
```

Comment:
Why don't you use css-vars?
```suggestion
                  ...viewContentStyles({
                    width: ctx.spy(width), height: ctx.spy(height)
                  }),
```

```ts
const viewContentStyles = styled('')`
  overflow-y: scroll;
  width: ${({ width: number }) => width}px;
  height: ${({ height: number }) => height}px;
`
```